jenkins_cli/cli.py

```diff
@@ -71,6 +72,22 @@ def xml_to_string(root):
     return ElementTree.tostring(root, encoding=('unicode' if sys.version_info[0] == 3 else None))
 
 
+def check_nonnegative(value):
```

Comment:
Fair point. It makes sense to separate logically different parts of code. So I moved the argument parsing logic to a separate file (`cli_arguments.py`), **including** the parts in `__init__.py`, so the entry point is fairly simple now. Is the current state ok with you?